src/components/Breadcrumb/Breadcrumb.js

```diff
@@ -119,7 +119,7 @@ fabric.Breadcrumb.prototype = (function() {
   var _addBreadcrumbItems = function(maxItems) {
     _resetList(_breadcrumbList);
     var i = _itemCollection.length - maxItems;
-
+    i=i<0? 0 : i;
```

Comment:
Nit. That'd be great if you could put spaces between the variables, values and operators in the first part of the ternary operation.